scripts/compare-metrics/compare-metrics.R

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env Rscript
+
+# get script location and activate renv
+script_dir <- here::here("scripts/compare-metrics")
+renv::load(script_dir)
+
+# parse options
+library(optparse)
+
+option_list <- list(
+  make_option(
+    c("-r", "--ref_s3"),
+    type = "character",
+    default = "s3://nextflow-ccdl-results/scpca-prod",
+    help = "S3 URI for the bucket and prefix for the reference files"
+  ),
+  make_option(
+    c("-c", "--comp_s3"),
+    type = "character",
+    default = "s3://nextflow-ccdl-results/scpca-staging",
+    help = "S3 URI for the bucket and prefix for the comparison S3 files"
+  ),
+  make_option(c("-p", "--project_id"),
+    type = "character",
+    default = "all",
+    help = "Project ID(s) to filter the metrics files. Default is all projects"
+  ),
+  make_option(
+    c("-o", "--output_file"),
+    type = "character",
+    default = "compare-metrics.html",
+    help = "Output file name for the rendered report"
+  ),
+  make_option(
+    c("--skip_download"),
```

Comment:
```suggestion
    c("--use_cache"),
```
Maybe we can be more explicit about what this option is doing? I might also mirror this name change in the template.

Reply:
Also since you noted saving the rds file is temporary in the template notebook, does that mean this argument is temporary? If so, let's note that so we remember to remove it. 
Alternatively, we could keep it but ensure that when it runs in a workflow we do a fresh download of the metrics files.